lib/src/database/sqflite_box.dart

```diff
@@ -110,6 +112,9 @@ class BoxCollection {
       );
 
   Future<void> close() => _db.close();
+
+  Future<void> delete() =>
+      (_factory ?? databaseFactory).deleteDatabase(_db.path);
```

Comment:
I was not sure about this 🤔 and had a hard time finding information but since 2019 it looks like that sqflite is force closing the database always by itself before deleting it: https://github.com/tekartik/sqflite/issues/213

Reply:
Oh, they know how people will misuse the API, I see!